packages/platform-sdk/src/coins/network.models.ts

```diff
@@ -17,158 +17,160 @@ export type CoinTransactionTypes =
 
 export type ExpirationType = "height" | "timestamp";
 
-export interface SignatureMethods {
+export interface SignatureMethod {
 	default: boolean;
-	ledgerS?: boolean;
-	ledgerX?: boolean;
+	ledgerS: boolean;
+	ledgerX: boolean;
 }
 
-export interface NetworkFeatureFlags {
-	Client?: {
-		transaction?: boolean;
-		transactions?: boolean;
-		wallet?: boolean;
-		wallets?: boolean;
-		delegate?: boolean;
-		delegates?: boolean;
-		votes?: boolean;
-		voters?: boolean;
-		configuration?: boolean;
-		fees?: boolean;
-		syncing?: boolean;
-		broadcast?: boolean;
-	};
-	Fee?: {
-		all?: boolean;
-	};
-	Identity?: {
-		address?: {
-			mnemonic?: boolean;
-			multiSignature?: boolean;
-			publicKey?: boolean;
-			privateKey?: boolean;
-			wif?: boolean;
-			secret?: boolean;
-			validate?: boolean;
-		};
-		publicKey?: {
-			mnemonic?: boolean;
-			multiSignature?: boolean;
-			wif?: boolean;
-			secret?: boolean;
-		};
-		privateKey?: {
-			mnemonic?: boolean;
-			wif?: boolean;
-			secret?: boolean;
-		};
-		wif?: {
-			mnemonic?: boolean;
-			secret?: boolean;
-		};
-		keyPair?: {
-			mnemonic?: boolean;
-			privateKey?: boolean;
-			wif?: boolean;
-			secret?: boolean;
-		};
-	};
-	Ledger?: {
-		getVersion?: boolean;
-		getPublicKey?: boolean;
-		signTransaction?: boolean;
-		signMessage?: boolean;
-	};
-	Link?: {
-		block?: boolean;
-		transaction?: boolean;
-		wallet?: boolean;
-	};
-	Message?: {
-		sign?: boolean;
-		verify?: boolean;
-	};
-	Peer?: {
-		search?: boolean;
-	};
-	Transaction?: {
-		transfer?: SignatureMethods;
-		secondSignature?: SignatureMethods;
-		delegateRegistration?: SignatureMethods;
-		vote?: SignatureMethods;
-		multiSignature?: SignatureMethods;
-		ipfs?: SignatureMethods;
-		multiPayment?: SignatureMethods;
-		delegateResignation?: SignatureMethods;
-		htlcLock?: SignatureMethods;
-		htlcClaim?: SignatureMethods;
-		htlcRefund?: SignatureMethods;
-	};
-	Miscellaneous?: {
-		dynamicFees?: boolean;
-		memo?: boolean;
-		utxo?: boolean;
-	};
-	Derivation?: {
-		bip39?: boolean;
-		bip44?: boolean;
-		bip49?: boolean;
-		bip84?: boolean;
-		secret?: boolean;
-	};
-	Internal?: {
-		fastDelegateSync?: boolean;
-	};
+export interface NetworkHost {
+	type: "full" | "musig" | "archival" | "explorer";
+	host: string;
+}
+
+export interface ImportMethod {
+	default: boolean;
+	ability: { read: boolean; write: boolean };
+}
+
+export interface MnemonicMethod {
+	bip39?: boolean;
+	bip44?: boolean;
+	bip49?: boolean;
+	bip84?: boolean;
 }
 
 export interface NetworkManifest {
-	id: string;
+	uuid: string;
 	type: string;
```

Comment:
I wouldn't call this uuid, as they have a very specific format and it's not what I see above

Reply:
https://github.com/ArkEcosystem/platform-sdk/pull/1571#discussion_r636848607